tests/unit/job/test_query.py

```diff
@@ -989,6 +989,40 @@ def test_result(self):
             [query_results_call, query_results_call, reload_call, query_page_call]
         )
 
+    def test_result_dry_run(self):
+        query_resource_done = {
```

Comment:
Is this resource actually needed? I don't think we need to simulate `jobs.query` or `getQueryResults` in this test.